lib/engine/game/g_1822_africa/step/token.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require_relative '../../g_1822/step/token'
+
+module Engine
+  module Game
+    module G1822Africa
+      module Step
+        class Token < G1822::Step::Token
+          def actions(entity)
+            actions = super.dup
+
+            actions << 'choose_ability' if need_choose_ability?(entity, actions) && !actions.include?('choose_ability')
+
+            actions << 'pass' if actions.any? && !actions.include?('pass')
```

Comment:
The JS produced by Opal for `any?` without a block is much less efficient than `!empty?`. (Unfortunately there are also a bunch of other places in the code doing this that need to be cleaned up too)

Reply:
Can we make rubicon check for that?

Reply:
Not with any cops/configurations I’ve been able to find

Reply:
@michaeljb changed it